### PR TITLE
Implement locale

### PR DESCRIPTION
**Pull request by sirjla**
# Resumen
Se implemento locale
# Detalles
Se implemento locale en ingles y español para todas las vistas

### CODE REVIEW

File: src/components/LeadTimeChart.vue [hunk 1]
```diff
@@ -1,27 +1,27 @@
...
     </div>
     <div v-if="deployTime !== '0'" class="category-container" :style="deployWidth">
       <div class="category-line category-line-bottom"></div>
-      <p class="category-text">Deploy Time</p>
+      <p class="category-text">{{ $t('leadTime.deployTime') }}</p>
     </div>
     <div class="category-container" :style="leadWidth">
       <div class="category-line category-line-bottom"></div>
-      <p class="category-text">Lead Time ({{ leadTime.toLocaleString({maximumFractionDigits: 2}) }})</p>
+      <p class="category-text">{{ $t('leadTime.leadTime') }} ({{ leadTime.toLocaleString({maximumFractionDigits: 2}) }})</p>
```
Comment (monkeyci) [inline src/components/LeadTimeChart.vue]: Line 24 exceeds the maximum line length of 120  max-len

File: src/components/LeadTimeChart.vue [hunk 2]
```diff
@@ -1,27 +1,27 @@
 <template>
   <div class="stacked-bar-graph">
     <div class="category-container">
-      <p class="category-text">Time To Live ({{ timeToLive.toLocaleString({maximumFractionDigits: 2}) }})</p>
+      <p class="category-text">{{ $t('leadTime.timeToLive') }} ({{ timeToLive.toLocaleString({maximumFractionDigits: 2}) }})</p>
```
Comment (monkeyci) [inline src/components/LeadTimeChart.vue]: Line 4 exceeds the maximum line length of 120   max-len

File: src/components/BoardWizard.vue [hunk 3]
```diff
@@ -138,7 +138,7 @@
         </div>
       </div>
       <div class="wizard--button-container">
-        <button id="back_button" class="button button-disabled" v-on:click="stepBack" disabled>BACK</button>
+        <button id="back_button" class="button button-disabled" v-on:click="stepBack" disabled>{{ $t('wizard.general.back') }}</button>
```
Comment (monkeyci) [inline src/components/BoardWizard.vue]: Line 141 exceeds the maximum line length of 120  max-len

**Comment by sirjla** [comment]
[Añadir Locale](https://trello.com/c/oNcRjxq3/94-a%C3%B1adir-locale)

**Pull request merged**